boefjes/boefjes/app.py

```diff
@@ -93,28 +94,38 @@ def _fill_queue(self, queue_type: WorkerManager.Queue) -> None:
         logger.debug("Popping from queue %s", queue_type.value)
 
         try:
-            p_item = self.scheduler_client.pop_item(queue_type.value)
+            p_items = self.scheduler_client.pop_items(queue_type.value)
         except (HTTPError, ValidationError):
             logger.exception("Popping task from scheduler failed, sleeping %s seconds", self.settings.poll_interval)
             time.sleep(self.settings.poll_interval)
             return
 
-        if p_item is None:
+        if not p_items:
             time.sleep(self.settings.poll_interval)
             return
 
-        logger.info("Handling task[%s]", p_item.data.id)
+        for p_item in p_items:
+            logger.info("Handling task[%s]", p_item.data.id)
```

Comment:
Seems a bit odd imho to loop only for logging. Isn't this information already output below?